test/classes/initializers/compilerGenerated/generics/withParamTypeNoDef.chpl

```diff
@@ -2,7 +2,7 @@ class Foo {
   param p: bool;
 }
 
-var foo = new Foo(true);
+var foo = new borrowed Foo(true);
```

Comment:
'foo' is still deleted, is that acceptable?

Reply:
Good catch. `delete foo` should be a compiler error, right?

Reply:
Yeah, in the future, not yet. We have to migrate tests over before we can make that an error. But it's right that it shouldn't be deleted later.